tests/providers/amazon/aws/hooks/test_batch_waiters.py

```diff
@@ -305,13 +305,13 @@ def test_aws_batch_job_waiting(aws_clients, aws_region, job_queue_name, job_defi
     assert "Waiter JobComplete failed: Max attempts exceeded" in str(err.value)
 
     # wait for job to be running (or complete)
-    job_running_waiter.config.delay = 0.25  # sec delays between status checks
-    job_running_waiter.config.max_attempts = 50
+    job_running_waiter.config.delay = 1  # sec delays between status checks
+    job_running_waiter.config.max_attempts = 100
     job_running_waiter.wait(jobs=[job_id])
 
     # wait for job completion
-    job_complete_waiter.config.delay = 0.25
-    job_complete_waiter.config.max_attempts = 50
+    job_complete_waiter.config.delay = 1
```

Comment:
Same here.

Reply:
Yes, I'm trying to fiddle with these values to see if it resolves the last test. Not sure what else to try :)